hazelcast/src/main/java/com/hazelcast/dataconnection/impl/JdbcDataConnection.java

```diff
@@ -99,11 +101,13 @@ public List<DataConnectionResource> listResources() {
             ResultSet tables = metaData.getTables(null, null, "%", null);
             List<DataConnectionResource> result = new ArrayList<>();
             while (tables.next()) {
-                result.add(new DataConnectionResource(
-                        "TABLE",
-                        // TODO quoting? Using String[]?
-                        tables.getString("TABLE_SCHEM") + "." + tables.getString("TABLE_NAME")
-                ));
+                String[] name = Stream.of(tables.getString("TABLE_CAT"),
+                                              tables.getString("TABLE_SCHEM"),
```

Comment:
TABLE_SCHEM**A** ?

Reply:
It is indeed `TABLE_SCHEM`

See 
https://docs.oracle.com/javase/8/docs/api/java/sql/DatabaseMetaData.html#getTables-java.lang.String-java.lang.String-java.lang.String-java.lang.String:A-